php-fpm/recipes/configure.rb

```diff
@@ -52,7 +50,13 @@
   Chef::Log.info("Adding to php sendmail stmt: #{sendmail_params}")
 end
 
-template "#{etc_fpm_dir}/#{conf_fpm}" do
+execute 'update-alternatives' do
+  command '/usr/bin/update-alternatives --install /usr/sbin/php-fpm php-fpm /usr/sbin/php-fpm5.6 0'
+  creates '/usr/sbin/php-fpm'
```

Comment:
This implies that it only runs if `/usr/sbin/php-fpm` does not exist - wouldnt it be cleaner to update anyway?